### PR TITLE
78526 vulnerabilities fix in pipeline

### PR DESCRIPTION
**Pull request by Shirin14926**
Vulnerabilities observed in dev when Nuget restore command was added.
We have suppressed few, few are resolved by updating stable versions of nuget packages.

### CODE REVIEW

**Review by harshal11869 — APPROVED** [review]
Looks Good.

**Review by jrippington — APPROVED** [review]
Looks good, thanks!

File: NVDSuppressions.xml [hunk 1]
```diff
@@ -10,4 +10,284 @@
...
+		<cve>CVE-2021-24112</cve>
+	</suppress>
+	<suppress>
+		<notes>
+			<![CDATA[
+      file name: Microsoft.AspNetCore.Hosting.Abstractions.dll
+        See https://github.com/dotnet/announcements/issues/112 for full MS Details on the CVE
+          A denial of service vulnerability exists when .NET Framework or .NET Core improperly handle web requests, aka '.Net Framework and .Net Core Denial of Service Vulnerability'. This CVE ID is unique from CVE-2019-0820, CVE-2019-0981.
+          Vunerability applies to System.Private.Uri version	4.3.0 and 4.3.1. System.Private.Uri is part of the .NetCore runtime, and fixed with version System.Private.Uri version 4.3.2.
+        FSS is pinned to .Net core SDK > 3.1.408 which is not vunerable, and deployed to Azure App Service which will have the security patch applied.
```
Comment (martyn-fewtrell) [inline NVDSuppressions.xml:32]: FSS is pinned?

Assume ERP is 6.......

File: NVDSuppressions.xml [hunk 2]
```diff
@@ -10,4 +10,284 @@
...
+	<suppress>
+		<notes>
+			<![CDATA[
+      file name: Microsoft.AspNetCore.Hosting.Abstractions.dll
+        See https://github.com/dotnet/announcements/issues/113 for full MS Details on the CVE
+        Vunerability applies to System.Private.Uri version	4.3.0 and 4.3.1. System.Private.Uri is part of the .NetCore runtime, and fixed with version System.Private.Uri version 4.3.2.
+        The issue is fixed:         
+          2.1.11 or SDK 2.1.507 for Visual Studio 2017 or SDK 2.1.604 for Visual Studio 2019
+          2.2.5 or SDK 2.2.107 for Visual Studio 2017 or SDK 2.2.204 for Visual Studio 2019
+        ERP Facade is pinned to .Net core SDK > 3.1.408 which is not vunerable, and deployed to Azure App Service which will have the security patch applied.
```
Comment (martyn-fewtrell) [inline NVDSuppressions.xml:46]: ? ERP Facade is pinned to .Net core SDK > 6.0.101

File: NVDSuppressions.xml [hunk 3]
```diff
@@ -10,4 +10,284 @@
...
+		</notes>
+		<cve>CVE-2019-0981</cve>
+	</suppress>
+	<suppress>
+		<notes>
+			<![CDATA[
+      file name: Microsoft.AspNetCore.Hosting.Abstractions.dll
+        See https://github.com/dotnet/announcements/issues/121 for full MS Details on CVE.
+        A denial of service vulnerability exists when .NET Core improperly handles web requests. An attacker who successfully exploited this vulnerability could cause a denial of service against a .NET Core web application. The vulnerability can be exploited remotely, without authentication.
+      This CVE applies to any .NET Core based application running on .NET Core 2.1.0-2.1.12, or .NET Core 2.2.0-2.2.6. FSS is pinned to .Net core SDK > 3.1.408 which is not vunerable, and deployed to Azure App Service which will have the security patch applied.
```
Comment (martyn-fewtrell) [inline NVDSuppressions.xml:57]: ERP Facade is pinned to .Net core SDK > 6.0.101

File: NVDSuppressions.xml [hunk 4]
```diff
@@ -10,4 +10,284 @@
...
+    ]]>
+		</notes>
+		<cve>CVE-2019-1301</cve>
+	</suppress>
+	<suppress>
+		<notes>
+			<![CDATA[
+   file name: Microsoft.AspNetCore.Hosting.Abstractions.dll
+    An information disclosure vulnerability exists in .NET Framework and .NET Core which allows bypassing Cross-origin Resource Sharing (CORS) configurations, aka ".NET Framework Information Disclosure Vulnerability."
+    This applies to version of System.Net.Http as deployed as part of .Net core below 2.1.7 or 2.2.1. ERP Facade is pinned to .Net core SDK > 3.1.408 which is not vunerable, and deployed to Azure App Service which will have the security patch applied.
```
Comment (martyn-fewtrell) [inline NVDSuppressions.xml:67]: ERP Facade is pinned to .Net core SDK > 6.0.101

File: NVDSuppressions.xml [hunk 5]
```diff
@@ -10,4 +10,284 @@
...
+	</suppress>
+	<suppress>
+		<notes>
+			<![CDATA[
+   file name: Microsoft.AspNetCore.Hosting.Abstractions.dll
+    See https://github.com/dotnet/announcements/issues/111 for MS details on this CVE.
+        Package name: System.Text.RegularExpressions.dll (included as part of the .Net Core runtime)
+      	Vulnerable versions: 4.3.0 and below
+        Secure versions: 4.3.1+
+    This CVE applies to .Net Core versions below 2.2.107 (runtime 2.2.5). ERP Facade is pinned to .Net core SDK > 3.1.408 which is not vunerable, and deployed to Azure App Service which will have the security patch applied.
```
Comment (martyn-fewtrell) [inline NVDSuppressions.xml:80]: ERP Facade is pinned to .Net core SDK > 6.0.101

File: NVDSuppressions.xml [hunk 6]
```diff
@@ -10,4 +10,284 @@
...
+   ]]>
+		</notes>
+		<cve>CVE-2019-0820</cve>
+	</suppress>
+	<suppress>
+		<notes>
+			<![CDATA[
+   file name: Microsoft.AspNetCore.Hosting.Abstractions.dll
+    See https://github.com/dotnet/announcements/issues/97 for MS details on this CVE.
+    This CVE only applies to .NetCore versions below 2.1.8 and 2.2.2. ERP Facade is pinned to .Net core SDK > 3.1.408 which is not vunerable, and deployed to Azure App Service which will have the security patch applied.
```
Comment (martyn-fewtrell) [inline NVDSuppressions.xml:90]: ERP Facade is pinned to .Net core SDK > 6.0.101

**Review by martyn-fewtrell — CHANGES_REQUESTED** [review]
I realise this is Symantec's but it would be good to be consistent with which version we are pinned to otherwise it looks like it is out of date and would need reviewing again

File: NVDSuppressions.xml (hunk 6 above)
Comment (Shirin14926) [reply NVDSuppressions.xml:90]: Done, Thanks

File: NVDSuppressions.xml (hunk 5 above)
Comment (Shirin14926) [reply NVDSuppressions.xml:80]: Done, Thanks

File: NVDSuppressions.xml (hunk 4 above)
Comment (Shirin14926) [reply NVDSuppressions.xml:67]: Done, Thanks

File: NVDSuppressions.xml (hunk 3 above)
Comment (Shirin14926) [reply NVDSuppressions.xml:57]: Done, Thanks

File: NVDSuppressions.xml (hunk 2 above)
Comment (Shirin14926) [reply NVDSuppressions.xml:46]: Missed it, Thanks

**Review by martyn-fewtrell — APPROVED** [review]
Thanks you - Looks good

**Pull request merged**